package.json

```
@@ -1,57 +1,25 @@
 {
-  "name": "linaria",
-  "version": "2.0.0",
-  "description": "Blazing fast zero-runtime CSS in JS library",
-  "main": "lib/index.js",
-  "module": "esm/index.js",
-  "bin": "bin/linaria.js",
-  "files": [
-    "bin/",
-    "lib/",
-    "esm/",
-    "react/",
-    "*.d.ts",
-    "babel.js",
-    "evaluators.js",
-    "loader.js",
-    "rollup.js",
-    "server.js",
-    "stylelint-config.js"
-  ],
+  "private": true,
   "license": "MIT",
   "repository": "git@github.com:callstack/linaria.git",
   "bugs": {
     "url": "https://github.com/callstack/linaria/issues"
   },
   "homepage": "https://github.com/callstack/linaria#readme",
-  "keywords": [
-    "react",
-    "css",
-    "css-in-js",
-    "styled-components",
-    "babel-plugin",
-    "babel",
-    "webpack"
-  ],
   "scripts": {
+    "prepare": "yarn lerna run prepare",
+    "test": "yarn lerna run test",
+    "test:coverage": "yarn lerna run test -- -- --coverage",
```

Do we really need these extra `--`?
```suggestion
    "test:coverage": "yarn lerna run test --coverage",
```

---

Unfortunately. The first one says that the rest of arguments should be ignored by yarn. Usually, it can be simply ignored, since yarn passes by all unknown arguments, but the following `--` is not an unknown.